src/inc/winwrap.h

```diff
@@ -48,6 +47,7 @@
 #include <specstrings.h>
 
 #include "registrywrapper.h"
+#include "winwrappers.h"
```

Comment:
The generic nature is by design. We could use this to wrap around other logic when needed